recipe/bld.bat

```diff
@@ -39,6 +39,7 @@ cmake -G "Ninja" ^
       -DARROW_ORC:BOOL=ON ^
       -DARROW_PACKAGE_PREFIX="%LIBRARY_PREFIX%" ^
       -DARROW_PARQUET:BOOL=ON ^
+      -DPARQUET_BUILD_EXECUTABLES:BOOL=ON ^
```

Comment:
Could also pass `-DARROW_BUILD_UTILITIES=ON` (same in `build.sh`)

Reply:
Good point! I'll wait for the maintainers if they want one or two packages for the parquet/arrow executables.